bookwyrm/templates/book/edit/edit_book_form.html

```diff
@@ -119,10 +119,16 @@ <h2 class="title is-4">{% trans "Authors" %}</h2>
                 </fieldset>
                 {% endif %}
                 <div class="field">
-                    <label class="label" for="id_add_author">{% trans "Add Authors:" %}</label>
-                    <input class="input" type="text" name="add_author" id="id_add_author" placeholder="{% trans 'John Doe, Jane Smith' %}" value="{{ add_author }}" {% if confirm_mode %}readonly{% endif %}>
-                    <span class="help">{% trans "Separate multiple values with commas." %}</span>
+                    <label class="label">{% trans "Add Authors:" %}</label>
+                    {% for author in add_author %}
+                    <label class="label is-sr-only" for="id_add_author{% if not forloop.first %}-{{forloop.counter}}{% endif %}">{% trans "Add Author" %}</label>
+                    <input class="input" type="text" name="add_author" id="id_add_author{% if not forloop.first %}-{{forloop.counter}}{% endif %}" placeholder="{% trans 'Jane Doe' %}" value="{{ author }}" {% if confirm_mode %}readonly{% endif %}>
+                    {% empty %}
+                    <label class="label is-sr-only" for="id_add_author">{% trans "Add Author" %}</label>
+                    <input class="input" type="text" name="add_author" id="id_add_author" placeholder="{% trans 'Jane Doe' %}" value="{{ author }}" {% if confirm_mode %}readonly{% endif %}>
+                    {% endfor %}
                 </div>
+                <span class="help"><a id="another_author_field" onclick="BookWyrm.duplicateInput('id_add_author')">{% trans "Add Another Author" %}</a></span>
```

Comment:
I think this makes more semantic as a `<button type="button" class="button is-small" ...>` than an `a`

Reply:
It would be more consistent with the rest of the javascript to use a data attribute or class to indicate that this is a duplication button, and add the listener in the javascript file, rather than using `onclick`. Something like:

``` javascript
document.querySelectorAll('[data-duplicate]')
    .forEach(t => t.addEventListener('click', this.duplicateInput.bind(this)));
```

Reply:
That makes total sense. I can get to it in the next couple of days.